vllm/v1/engine/core.py

```diff
@@ -32,7 +32,7 @@
 
 POLLING_TIMEOUT_MS = 5000
 POLLING_TIMEOUT_S = POLLING_TIMEOUT_MS // 1000
-LOGGING_TIME_S = 5000
+LOGGING_TIME_S = POLLING_TIMEOUT_S
```

Comment:
Do we need to change this? A little bit unintuitive on why

Reply:
Previously it was 5000S, this just switches t to be 5s

Reply:
yeah maybe just hard code this?

Reply:
I think Simon's question is why we're tying the them when they seem unrelated to each other

Reply:
Oh - yeah np will revert.